cluster/addons/dns/skydns-rc.yaml.base

```diff
@@ -16,25 +16,22 @@
 
 # __MACHINE_GENERATED_WARNING__
 
-apiVersion: v1
-kind: ReplicationController
+apiVersion: extensions/v1beta1
+kind: Deployment
 metadata:
-  name: kube-dns-v20
+  name: kube-dns
   namespace: kube-system
   labels:
     k8s-app: kube-dns
-    version: v20
     kubernetes.io/cluster-service: "true"
 spec:
-  replicas: __PILLAR__DNS__REPLICAS__
```

Comment:
Where does the initial replica count live now?

Maybe leave a comment here specifically saying `# replicas: not specified here, see xyz.yaml` or maybe even specify why.